### PR TITLE
mtuner: Add version 4.4.8

### LINKED ISSUE #5356: [Request] Add MTuner
MTuner is a C/C++ memory profiler and memory leak finder for Windows, PlayStation 4 and 3, Android and other platforms.

github project: https://github.com/milostosic/mtuner

### PR DESCRIPTION
**Pull request by zzfnohell**
mtuner: Add version 4.4.8

Closes #5356

### CODE REVIEW

**Comment by pratikpc** [comment]
@zzfnohell is cv2pdb popular among mtuner users?

> Users can convert DMD CodeView/DWARF debug symbols to PDB format using cv2pdb.

@zzfnohell also can you create a sort of short video which showcases that the Scoop install is working

File: bucket/mtuner.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "homepage": "https://github.com/milostosic/MTuner",
+    "license": "BSD-2-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/milostosic/MTuner/releases/download/v4.4.8/mtuner64.zip",
+            "hash": "a33785dcb9cc42f90dc963790f9aee4130dea4f654ccfdbd77f622bc9ff7e762"
+        }
+    },
+    "extract_dir": "MTuner",
+    "bin": "MTuner.exe",
```
Comment (rashil2000) [inline bucket/mtuner.json:13]:
```suggestion
    "bin": "MTuner.com",
```

File: bucket/mtuner.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "4.4.8",
+    "description": "MTuner is a C/C++ memory profiler and memory leak finder for Windows, PlayStation 4 and 3, Android and other platforms.",
```
Comment (rashil2000) [inline bucket/mtuner.json:3]:
```suggestion
    "description": "A C/C++ memory profiler and memory leak finder for Windows, PlayStation 4 and 3, Android and other platforms.",
```

**Comment by zzfnohell** [comment]
> @zzfnohell is cv2pdb popular among mtuner users?
> 
> > Users can convert DMD CodeView/DWARF debug symbols to PDB format using cv2pdb.
> 

Sorry, The question is too difficult for me to answer.
I'm not an expert on mtuner. I found it on google.
In my understanding, it depends on which toolchain(or platform) is used, pdb files are created by default when using visual studio toolchain, other toolchains(gcc/clang) use different debug symbol formats, cv2pdb is suitable for this situation.


> @zzfnohell also can you create a sort of short video which showcases that the Scoop install is working

https://user-images.githubusercontent.com/1160608/142646928-d5ed9cf7-ea4a-4864-8c46-e89761f91be5.mp4

**Comment by pratikpc** [comment]
@zzfnohell installation works!!! This is nice.

Could you also validate mtuner works? For your personal projects with some testing.

Manifest looks fine but its safest for us to validate it all before we add.

@zzfnohell also nothing to persist correct?

**Comment by zzfnohell** [comment]
> Could you also validate mtuner works? For your personal projects with some testing.
A simple test project:
https://user-images.githubusercontent.com/1160608/146568348-9f8073f5-f7fe-4d12-9222-cea3f71d4272.mp4

> @zzfnohell also nothing to persist correct?
There is nothing to persist.

**Pull request merged**